docs/index.md

```diff
@@ -5,19 +5,55 @@ _navPath: toc.html
 _navRel: ../toc.html
 ---
 
-## Getting started
+.NET Documentation
+==================
 
-In this docset, you will find tutorials that will teach you how to get started with .NET Core. You can also find the API reference documentation. 
+Welcome to .NET! You can build any kind of app you want with .NET, from cloud to mobile to desktop to IoT to games. You can start building your next app today, for free, on Windows, OS X and Linux.
 
-## What kinds of applications do you want to build?
+* [Web](http://docs.asp.net/)
```

Comment:
I'd reorder this list so that "Console Applications" and "Web" are adjacent, and "Mobile" and "UWP" are adjacent. They seem related a bit.